tests/warp-drive__experiments/package.json

```diff
@@ -17,7 +17,7 @@
   "scripts": {
     "build:tests": "IS_TESTING=true EMBER_CLI_TEST_COMMAND=true ember build --output-path=dist-test --suppress-sizes",
     "_build:production": "bun run build:tests -e production",
-    "lint": "eslint . --quiet --cache --cache-strategy=content",
```

Comment:
accidental check-in?

Reply:
nay -- eslint's cache being buggy is why ci and editor and cli are sometimes not in agreement